frontend/src/common/components/chip/OnRouteBCChip.tsx

```diff
@@ -0,0 +1,22 @@
+import "./OnRouteBCChip.scss";
```

Comment:
in `Breadcrumb.scss` the reference was updated to `orbcStyles` without the `.scss` but here you leave in the `.scss`. Maybe add it back into Breadcrumb for consistency?